src/components/SearchBar/index.js

```diff
@@ -0,0 +1,39 @@
+import algoliasearch from "algoliasearch/lite";
+import { createRef, default as React, useState, useContext } from "react";
+import { InstantSearch } from "react-instantsearch-dom";
+import { ThemeContext } from "../ThemeContext";
+import StyledSearchBox from "./StyledSearchBox";
+import StyledSearchResult from "./StyledSearchResult";
+import StyledSearchRoot from "./StyledSearchRoot";
+import useClickOutside from "./useClickOutside";
+
+const SearchBar = ({ indices }) => {
+  const { colorMode } = useContext(ThemeContext);
```

Comment:
Mentioned this in the other file, but no reason this should be used outside of controlling the state of the theme